vincentvanbot/frompicture/predict.py

```diff
@@ -1,57 +1,37 @@
+"""This file is used to predict paintings that are closest to the given picture."""
+
 import os
-from sklearn.neighbors import NearestNeighbors
 import joblib
-import pandas as pd
 from google.cloud import storage
+import pandas as pd
 from vincentvanbot.params import BUCKET_NAME, BUCKET_INITIAL_DATASET_FOLDER
-from vincentvanbot.preprocessing.utils import get_jpg_link
+from vincentvanbot.utils import get_jpg_link
 
-def train_model(df_transformed):
-    """Takes preprocessed train data as df. Returns fitted KNN model 
-    and train data image indexes (used then to refer back to initial database).
-    Saves locally model and indexes."""
-    knn_model = NearestNeighbors().fit(df_transformed)
-    
-    joblib.dump(knn_model,'model.joblib')
-    joblib.dump(df_transformed.index,'train_indexes.joblib')
-
-def save_model_to_cloud(rm=False):
-    """Uploads fitted model and related indexes to GCloud."""
-    client = storage.Client().bucket(BUCKET_NAME)
-    
-    for filename in ['model.joblib','train_indexes.joblib']:
-        storage_location = f"predict/{filename}"
-        blob = client.blob(storage_location)
-        blob.upload_from_filename(filename)
-        print(f"=> {filename} uploaded to bucket {BUCKET_NAME} inside {storage_location}")
-    if rm:
-        os.remove('model.joblib')
-        os.remove('train_indexes.joblib')
 
 def get_closest_images_indexes(user_input_transformed, nsimilar=3, rm=True):
     """Takes user_input_transformed as np.array. Downloads fitted knn model and related indexes.
     Returns indexes of nsimilar closest images"""
     client = storage.Client().bucket(BUCKET_NAME)
     
     # download model
-    local_name = 'model_10000.joblib'
+    local_name = 'model.joblib'
```

Comment:
Shouldn't we use method 'flat_images_db_download' from data.py here? also this function here doesn't cover the case when size is present in the filename